js/chessMoveReminder.js

```diff
@@ -27,71 +26,84 @@ function parseSecondsFromClock(clock) {
-                let currentClock = mutation.target.innerText;
-                clearTimeout(timer);
-                // For some reason chess.com/live#g=xxx uses clock-playerTurn
-                // whereas chess.com/game/live/xxx uses clock-player-turn
-                let isTurn = mutation.target.classList.contains('clock-playerTurn')
-                    || mutation.target.classList.contains('clock-player-turn');
-                if (!isTurn && audio !== undefined) {
-                    audio.pause();
-                }
-                if (isTurn) {
-                    chrome.storage.sync.get({
-                        'who': 'HikaruGotham',
-                        'number': 60,
-                        'type': 'seconds'
-                    }, function(data) {
+        observer = new MutationObserver(function (mutations) {
+            let mutation = mutations[0];
+            let currentClock =
+                mutation.target.querySelector(".time") || mutation.target;
+            clearTimeout(timer);
+            // For some reason chess.com/live#g=xxx uses clock-playerTurn
+            // whereas chess.com/game/live/xxx uses clock-player-turn
+            let isTurn =
+                mutation.target.classList.contains("clock-playerTurn") ||
+                mutation.target.classList.contains("clock-player-turn") ||
+                mutation.target.classList.contains("running");
+            if (!isTurn && audio !== undefined) {
+                audio.pause();
+            }
+            if (isTurn) {
+                chrome.storage.sync.get(
+                    {
+                        who: "HikaruGotham",
+                        number: 60,
+                        type: "seconds",
+                    },
+                    function (data) {
                         let who = data.who;
                         let number = data.number;
                         let type = data.type;
 
                         let timeToWait;
 
-                        if (type === 'percentage') {
-                            let seconds = parseSecondsFromClock(currentClock) * number * 10;
+                        if (type === "percentage") {
+                            let seconds =
+                                parseSecondsFromClock(currentClock) *
+                                number *
+                                10;
                             timeToWait = Math.max(seconds, 2000);
                         } else {
                             timeToWait = number * 1000;
                         }
                         timer = setTimeout(playAudio, timeToWait, who);
-                    });
-                }
-            });
+                    }
+                );
+            }
         });
 
         observer.observe(target, {
-            attributes:true,
-            attributeFilter:['class']
+            attributes: true,
+            attributeFilter: ["class"],
+            attributeOldValue: true,
```

Comment:
Is the old value being used somewhere?

Reply:
Sorry, was using it for debugging. I'll remove it.